doc/api/os.md

```diff
@@ -378,9 +378,8 @@ added: v0.3.3
 
 The `os.uptime()` method returns the system uptime in number of seconds.
 
-*Note*: Within Node.js' internals, this number is represented as a `double`.
-However, fractional seconds are not returned and the value can typically be
-treated as an integer.
+*Note*: the return value includes fractions of a second on Windows.
```

Comment:
I would phrase it as:
`-*Note*: On Windows (and maybe other platforms) the value returned includes fractions of seconds.`